.github/workflows/one-action-to-rule-them-all.yml

```diff
@@ -0,0 +1,28 @@
+name: AERIUS
+
+on:
+  push:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+  release:
+    types: [published]
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: aerius/github-actions/main-action@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+          NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+      - uses: aerius/github-actions/publish-docker-action@main
```

Comment:
How does this work? I see a `main-action` folder in our github-actions repository (https://github.com/aerius/github-actions/blob/main/main-action/action.yml), but not a `publish-docker-action`.

Reply:
About to push that one. :) Done!